Include a simple ant build file for the platform.jar part of the source. The next 
upstream version should already include this.

--- /dev/null
+++ b/contrib/platform/build.xml
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project name="platform" default="jar" basedir=".">
+
+  <property name="src" location="src"/>
+  <property name="build" location="build"/>
+  <property name="dist"  location="../../build/"/>
+
+  <path id="classpath">
+    <fileset dir="${dist}" includes="jna.jar"/>
+  </path>
+
+  <target name="init">
+    <mkdir dir="${build}"/>
+  </target>
+
+  <target name="compile" depends="init" description="compile the source " >
+    <javac srcdir="${src}" destdir="${build}" classpathref="classpath"/>
+  </target>
+
+  <target name="jar" depends="compile" description="jar" >
+    <jar jarfile="${dist}/jna-platform.jar" basedir="${build}"/>
+  </target>
+
+  <target name="javadoc">
+    <!-- we don't build the javadoc for now, this should all 
+         be sorted in the next upstream release. -->
+  </target>
+
+  <target name="clean" description="clean" >
+    <delete dir="${build}"/>
+    <delete file="${dist}/jna-platform.jar"/>
+  </target>
+</project>
+
